### DISCUSSION TITLE
What are the differences between version 6.x and 7.x?

### DISCUSSION THREAD

**Discussion by janswist** [gotenberg/gotenberg#509] [Q&A]
Can't find any document on what are the differences between 6.x and 7.x. Curious what has changes, what has improved (as in: is it processing faster/better? slightly different API?)

**Comment by dakur** [comment]
@janswist It's in Releases section: https://github.com/gotenberg/gotenberg/releases/tag/v7.0.0

You can get there with this link:

![image](https://user-images.githubusercontent.com/9862581/191898457-2718a341-ec42-4c5b-95f0-30823dff40b9.png)
